client/components/Reports/SummarizeTestPlanVersion.jsx

```diff
@@ -77,6 +79,24 @@ const SummarizeTestPlanVersion = ({ testPlanVersion, testPlanReports }) => {
                 return (
                     <Fragment key={testPlanReport.id}>
                         <h2>{getTestPlanTargetTitle(testPlanTarget)}</h2>
+                        <DisclaimerInfo
+                            title="Unapproved Report"
+                            messageContent={
+                                <>
+                                    The information in this report is generated
+                                    from candidate tests. Candidate aria-at
+                                    tests are in review by assistive technology
+                                    developers and lack consensus regarding:
+                                    <ol>
+                                        <li>
+                                            applicability and validity of the
+                                            tests, and
+                                        </li>
+                                        <li>accuracy of test results.</li>
+                                    </ol>
+                                </>
```

Comment:
Are these props needed given they are the defaults?

Reply:
No, it wouldn't be needed here as well. Missed that one.